waspc/data/Generator/templates/react-app/src/auth/pages/OAuthCallback.tsx

```diff
@@ -14,7 +14,7 @@ export function OAuthCallbackPage() {
   const { error, user } = useOAuthCallbackHandler();
   
   if (user !== undefined && user !== null) {
-    return <Redirect to="{= onAuthSucceededRedirectTo =}" />;
+    return <Navigate to="{= onAuthSucceededRedirectTo =}" replace />;
```

Comment:
`<Navigate />` replace `<Redirect />` and the default behaviour changed, we need to add `replace` explicitly.

Reply:
If you got links to the changelog that prescribes this, you can include it into these comments.

I'm not sure is it worth spending the time, but it would help me better judge the change's validity.

Reply:
https://reactrouter.com/en/main/upgrading/v5#use-usenavigate-instead-of-usehistory

> If you prefer to use a declarative API for navigation (ala v5's Redirect component), v6 provides a Navigate component.